system_tests/attempt_system_tests.py

```diff
@@ -26,17 +26,18 @@
 import subprocess
 import sys
 
+from run_system_test import FailedSystemTestModule
 from run_system_test import run_module_tests
 
 
 MODULES = (
```

Comment:
Was previously alphabetical. It's worth adding a source comment as to why the order is this way.